lib/ember-test-helpers/test-module.js

```diff
@@ -122,6 +123,14 @@ export default Klass.extend({
     }
   },
 
+  teardownSubject: function() {
+    var subject = this.cache.subject;
+
+    if (subject) {
+      Ember.run(subject, 'destroy');
```

Comment:
It is possible that `subject` does not have a `destroy` method so you will probably want to do something like:

```javascript
Ember.run(function() {
  Ember.tryInvoke(subject, 'destroy');
});
```